cookies/resources/bytes.py

```diff
@@ -0,0 +1,13 @@
+import urllib
+
+def main(request, response):
+    if "set" in request.GET:
+        response.headers.append(b"Set-Cookie", b"cookiesarebananas=\xFFtest\xFF")
+        response.content = "set"
+    elif "get" in request.GET:
+        response.content = urllib.quote(request.headers["Cookie"])
```

Comment:
This will throw in Python 3, since `quote` lives in `urllib.parse` now.

I don't even know what the plans are for Python3 and tests, so this may be a non-issue if there is no plan, but something to consider:

```python
try: 
    from urllib.parse import quote
except ImportError:
    from urllib import quote
```

And then calling `quote` directly.

Reply:
Please use `from six.moves.urllib.parse import quote` :)

Reply:
Thanks, I think I'll hold off on merging this pending #26190.

Reply:
@annevk I think it's safe to merge now.

Reply:
I hadn't made it P3-compatible yet. 😊

I have now, but it also seems like this hasn't been decided by the IETF WG yet.